code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -482,6 +482,9 @@ GLOBAL_LIST_INIT(abductor_recipes, list ( \
 //Metal Hydrogen
 GLOBAL_LIST_INIT(metalhydrogen_recipes, list(
 	new /datum/stack_recipe("incomplete servant golem shell", /obj/item/golem_shell/servant, req_amount=20, res_amount=1),
+	new /datum/stack_recipe("ancient armor", /obj/item/clothing/suit/armor/riot/knight/greyscale, req_amount = 8, res_amount = 1),
+	new /datum/stack_recipe("ancient helmet", /obj/item/clothing/head/helmet/knight/greyscale, req_amount = 5, res_amount = 1),
```

Comment:
why, those are autolathe recipes, just put the disk from lavaland in and print the armor out of this material

Reply:
I don't want to limit atmosians that have to wait/hope for mining to bring up a disk, so I made this

Reply:
at least make some unique sprites and make it be a different armor if you want to powercreep them

Reply:
lol powercreep, is not even that good of an armor, and you saw my sprites right?

Reply:
I kinda agree with fikou here, if they're gonna be knight armor it should be the same as any of the others

Reply:
i'll make a similar one to the knight armor and call it something like "Atmor"